Readme.md

```diff
@@ -12,4 +12,4 @@ which provides the `POST` endpoint for the Big Give's user-generated content API
 
 * Zip up the *contents* of the folder (not the whole folder), including `node_modules/`. This
   need not include normally-hidden files like `.git*` metadata.
-* Upload [here](https://eu-west-2.console.aws.amazon.com/lambda/home?region=eu-west-2#/functions/content-api-post-sandbox?tab=graph).
+* Upload [here](https://eu-west-1.console.aws.amazon.com/lambda/home?region=eu-west-1#/functions).
```

Comment:
Are you testing this with a temporary Lambda name? Might be good to keep this in explicitly so we don't mix up manual, temporary deployments with the Terraform-managed ones

Reply:
Oh I see, so manual temporary ones I need to create a new function to test - I saw the page was empty and thought this was the wrong link, I've reverted the change @NoelLH

Reply:
I've tried to create a temporary one, but I'm blocked with permissions error:
![image](https://user-images.githubusercontent.com/47176130/74144192-331f1580-4bf4-11ea-8596-70a259f5c04e.png)

Reply:
@tbg-jasonfung thinking about this more, as the change is not too drastic and we're now updating code that is live / used in Prod, it's probably better to get things through code review and then go straight to applying updates with Terraform in the normal, systematic way, with the staging `apply` and manual QA happening first. The Terraform user whose key you use will also have permissions to programmatically create and update Lambdas.